app/templates/gentelella/admin/event/tickets/discount_codes_create.html

```diff
@@ -261,8 +261,7 @@
 
         var host = (document.URL.split("//")[1].split("/"))[0];
         $("#code").change(function(){
-            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?code="+$(this).val());
+            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?discount_code="+$(this).val());
```

Comment:
@Jatinbalodhi96  This is for discount and similarly for access you have another link. How will this combine both access and discount  together in a single link?
like: "**acode=VIP&dcode=888**". You want them both on a single link.